app/js/data/pool.js

```
@@ -72,6 +74,70 @@ class Pool {
       this.spaces[slot].used = true;
     }
   }
+
+  // Re-generate pool spaces from loaded card data
+  regenerateFromCardData(cardSpaces) {
+    // Reset all entries to unused
+    this.contents.forEach(reservation => {
+      reservation.entry.used = false;
+    });
+
+    // For each card space (excluding FREE), find matching entry in contents and mark as used
+    let poolIndex = 0;
+    for (let cardIndex = 0; cardIndex < 25; cardIndex++) {
+      if (cardIndex === 12) continue; // Skip FREE space
+
+      const cardEntry = cardSpaces[cardIndex];
+
+      // Find matching entry in contents by text
+      const matchingReservation = this.contents.find(reservation =>
+        reservation.entry.text === cardEntry.text
+      );
+
+      if (matchingReservation) {
+        this.spaces[poolIndex] = matchingReservation.entry;
+        matchingReservation.entry.used = true;
+      } else {
+        console.warn(`Could not find matching entry for: ${cardEntry.text} - entry may have been removed from dataset`);
+        // Continue without this entry, it will be undefined in this.spaces[poolIndex]
```

I forgot that we had an "enabled" property on entries and I changed the text on some of them instead of disabling them and creating ones with the correct text. Kat calls this "retiring" or soft deleting an entry.  I plan on going back through the history and restoring the old entries in a disabled state. Once I do that, we can change this to throw instead of warn in a future change/PR. Then we won't have to worry about undefined spaces, but we will need to test restoring a card with disabled entries.

edit: no action needed here, just context for you